include/seadsa/AllocWrapInfo.hh

```diff
@@ -34,7 +34,7 @@ protected:
 
 public:
   static char ID;
-  AllocWrapInfo() : ModulePass(ID), m_tliWrapper(nullptr) {}
+  AllocWrapInfo() : ImmutablePass(ID), m_tliWrapper(nullptr) {}
```

Comment:
How does this work? I believe that for `ImmutablePass`, `runOnModule` is not called.

Reply:
The method `runOnModule` in `AllocWrapInfo` has still the keyword `override` so I'm assuming that it's being called. But I didn't check. I need to write some tests.